runners/libretro/build.sh

```diff
@@ -42,7 +42,7 @@ BuildRetroarch() {
     cd ${source_dir}
     SHALLOW_CLONE=1 ./libretro-fetch.sh retroarch
     cd ${source_dir}/retroarch
-    ./configure
+    ./configure --disable-ffmpeg
```

Comment:
What does this do? Will it remove functionality from Retroarch needed to play games?

Reply:
ffmpeg is in there for the video player core. Not needed to play games. I also just force-disabled QT too.

Reply:
That's very cool, we may be able to lighten up the lutris runtime with that!